earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -14,13 +14,14 @@ sudo yum install -y epel-release
 ### RHEL 7
 
 ```bash
+sudo subscription-manager repos --enable=rhel-7-server-optional-rpms
```

Comment:
I think this command will work on regular RHEL 7 instances, but it throws an error if you run it on AWS instances. For AWS this worked for me: `sudo yum-config-manager --enable rhel-7-server-rhui-optional-rpms`. We'll need to include both.

Reply:
I had to register with RedHat, using "sudo subscription-manager register", before I could install anything on RHEL 6 and 7 in AWS.  What error are you getting?

I went with subscription-manager because yum-config-manager wasn't installed by default on RHEL6 and 7 minimal install, so using it instead of yum-config-manager let the instructions be the same in and out of AWS.

Reply:
The rub is that registering with RedHat costs money on a production system, and AWS covers the cost for you if you use yum-config-manager instead of subscription-manager. That's why I think it's best to have both, even if you have to add a step to install yum-config-manager.